client/src/Home.js

```diff
@@ -1,9 +1,169 @@
+import './Home.scss';
+import { Link } from 'react-router-dom';
+import uberLogo from './images/Uber-Open-Source Logo.png';
+import DMLogo from './images/DevMission Logo.jpg';
+import C4SFLogo from './images/Code4SF Logo.png';
+
 function Home() {
   return (
-    <main className="container">
-      <h1>Home</h1>
-      <p>Tech Equity (Test)</p>
-    </main>    
+    <main>
+      {/* Heading */}
+      <div className="top">
+        <div className="container">
```

Comment:
Since this top area spans the entire width of the screen, there's no need for row/col classes here... instead, I put a container class INSIDE the top div so that the content gets centered appropriately